### PR TITLE
use reusable instead of useable

### PR DESCRIPTION
**Pull request by MirjamvT**
for more consistent use of the term reusable

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Most proprietary software systems currently used by public organizations do not
 We consider public code to have these core values:
 
 * Inclusive
-* Usable
+* reusable
```
Comment (bvhme) [inline introduction.md]: What about a capital here :)

```suggestion
* Reusable
```

**Comment by bvhme** [comment]
@nebster, @clausmullie and @ElenaFdR, I'm wondering what you are thinking about this, as it is quite a core change.

**Comment by ElenaFdR** [comment]
Is it worth putting this as an issue, rather than a PR? This also ties into #203 and #208.

**Comment by bvhme** [comment]
For now I think this is conflating two goals, namely: to make software that is easy to use and to make software that can be redeployed by different organisations.

I'm going to close this, but we have to be aware these two realities are existant.

**Pull request closed without merging**